pymobiledevice3/services/webinspector.py

```diff
@@ -191,7 +192,10 @@ def get_open_pages(self):
     def open_app(self, bundle: str, timeout: Union[float, int] = 3) -> Application:
         self.await_(self._request_application_launch(bundle))
         self.get_open_pages()
-        return self.await_(asyncio.wait_for(self._wait_for_application(bundle), timeout=timeout))
+        try:
+            return self.await_(asyncio.wait_for(self._wait_for_application(bundle), timeout=timeout))
+        except TimeoutError:
+            raise LaunchApplicationTimeoutError()
```

Comment:
Why is it better than just `TimeoutError`?

Reply:
changed